string post_processor
string program_name
string program_comment
---
# Success if empty
string error
